core/src/view/view.cpp

```diff
@@ -281,21 +281,18 @@ void View::updateMatrices() {
             break;
     }
 
+    m_viewProj = m_proj * m_view;
+    m_invViewProj = glm::inverse(m_viewProj);
+
+    m_normalMatrix = glm::transpose(glm::inverse(glm::mat3(m_view)));
```

Comment:
Did we figure this out why oblique axis modification to `m_view` should not be considered when calculating the normal matrix? As per what I was seeing, normals were calculated correctly with axis contributions!